src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/production/GregtechMetaTileEntityTreeFarm.java

```diff
@@ -378,6 +379,10 @@ public CheckRecipeResult process() {
                 duration = TICKS_PER_OPERATION;
                 calculatedEut = GT_Values.VP[tier];
 
+                for (Mode mode : Mode.values()) {
+                    useToolForMode(mode);
+                }
```

Comment:
This approach ignores whether current mode is supported or not. Which results in excessive damaging of tool. Check loop above.

```suggestion
                for (Mode mode : Mode.values()) {
                    if (outputPerMode.get(mode) != null) useToolForMode(mode);
                }
```